src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/charactercount/CharacterCount.scala

```diff
@@ -38,11 +39,20 @@ case class CharacterCount(
   formGroupClasses: String = "",
   classes: String = "",
   attributes: Map[String, String] = Map.empty,
-  countMessageClasses: String = ""
+  countMessageClasses: String = "",
+  charactersUnderLimitText: Option[TranslatableStrings] = None,
+  charactersAtLimitText: Option[String] = None,
+  charactersOverLimitText: Option[TranslatableStrings] = None,
+  wordsUnderLimitText: Option[TranslatableStrings] = None,
+  wordsAtLimitText: Option[String] = None,
+  wordsOverLimitText: Option[TranslatableStrings] = None,
+  textareaDescriptionText: Option[String] = None
 )
 
 object CharacterCount {
 
+  type TranslatableStrings = Map[String, String]
```

Comment:
Could we move and rename this to `uk.gov.hmrc.govukfrontend.views.viewmodels.i18n.PluralForms` to copy the name from https://github.com/alphagov/govuk-frontend/blob/d92a05aeea22236c00760feb5289f450ab9035c6/src/govuk/macros/i18n.njk#L13-L15 and because I think this will be reused

Reply:
guess we'll also need to add a type alias for it https://github.com/hmrc/play-frontend-hmrc/blob/273ef63bedf6cfc91efb82cee8a9b83d9e18f8a5/src/main/scala/uk/gov/hmrc/govukfrontend/views/Aliases.scala can you have a type alias pointing to another type alias?

Reply:
This is looking quite odd 😆 I have an object 

`package uk.gov.hmrc.govukfrontend.views.viewmodels.i18n

object PluralForms {
  type PluralForms = Map[String, String]
}`

Then I have an Alias in Aliases pointing to the type in the object

`import uk.gov.hmrc.govukfrontend.views.viewmodels.i18n.PluralForms

trait Aliases {
  type PluralForms = PluralForms.PluralForms`

It is happy with this